canonical-web-and-design/ubuntu.com
# Update /core page

## Done

- Update /core page
- Add `/smartstart` and `tutorials` to the nav 
- Smart start bubble moved [here](https://github.com/canonical-web-and-design/ubuntu.com/pull/9144)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/core
- Or view demo at: https://ubuntu-com-9142.demos.haus/core
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copydoc](https://docs.google.com/document/d/1QoM0Yqx4JfDJ2G3OPt5bds-ximG1jASmU_PWX4iff3E/edit#heading=h.cc4g322ve6sn) 


## Issue / Card

Fixes [#3643](https://github.com/canonical-web-and-design/web-squad/issues/3643) and [#3668](https://github.com/canonical-web-and-design/web-squad/issues/3668)
